tox.ini

```diff
@@ -88,27 +88,18 @@ commands =
     figure: python -c "import sunpy.tests.helpers as h; print(h.get_hash_library_name())"
     figure: {env:PYTEST_COMMAND} -m "mpl_image_compare" --mpl --remote-data=any --mpl-generate-summary=html --mpl-baseline-path=https://raw.githubusercontent.com/sunpy/sunpy-figure-tests/sunpy-master/figures/{envname}/ {posargs}
 
-[testenv:build_docs]
+[testenv:build_docs-{,nogallery}]
```

Comment:
can I suggest flipping this logic so it's `-gallery` to opt-in rather than opt out?

Reply:
Sure. I just did it like that for backwards compatibility.